### ISSUE TITLE
Fail to copy text from some menu-items with NVDA+NumPad5

### ISSUE THREAD

**Issue by nvaccessAuto** [nvaccess/nvda#3093]
**Reported by k_kolev1985 on 2013-03-19 11:15**
I've noticed that NVDA fails to copy the text from some selected menu-items with the command NVDA+NumPad5 pressed trice. Applications in witch this can be observed are: Cobian Backup, KMPlayer and the context menus in the Google Translate module (window) of Balabolka. There may be other applications in witch this can happen, but I don't know. As far as I know, all of the problematic applications are written in Delphi and this may be part of the reason why the issue occurs.

Steps to reproduce:
1. Open one of the following applications: KMPlayer, Cobian Backup, or the Google Translate window within Balabolka.
2. Invoke a context menu or a menu from the menu-bar of the application in question.
3. Select a menu-item from the opened menu.
4. Press NVDA+NumPad5 trice.
5. Open any text editor (e.g. Notepad) and execute the paste command (e.g. Ctrl+V) to paste the copied to the clipboard in step #4 text.
Expected result: NVDA should say that the text from the selected menu item is copied to the clipboard and you can than pasted without problems in the text editor.
Actual result: NVDA says that the text is copied, but upon pasting, there is nothing pasted.

Notes:
1. I can visually verify that the menu-item in question is selected and that upon pasting there is really nothing pasted.
2. I guess that an empty string is copied while performing the NVDA+NumPad5 command, because after it is executed, the clipboard is cleared (previously there was some text copied into it) but upon pasting after the NVDA+NumPad5 command, nothing is pasted at all with the normal paste command.

I'm using Windows 7 SP1 Ultimate 32-bit with bulgarian language pack and bulgarian locales. NVDA is main snapshot r5967. But I think this problem is not recent - it is from a long time.

**Comment by leonardder** [comment]
@k-kolev1985: Could you provide an update on this issue?

**Comment by k-kolev1985** [comment]
@leonardder This was a bug in NVDA and it was fixed a long time ago.

**Comment by ehollig** [comment]
Per https://github.com/nvaccess/nvda/issues/3093#issuecomment-316922219, closing
